tests/test_utils.py

```diff
@@ -129,16 +129,10 @@ def test_clean_prefix_map_strict(self):
     def test_clean_prefix_map_not_strict(self):
         """Test clean prefix map with 'strict'=False."""
         msdf = parse_sssom_table(f"{data_dir}/test_clean_prefix.tsv")
-        original_curie_map = msdf.prefix_map
```

Comment:
this is removed for the same reason as before, since a lot of extra content gets added during the new loading, and I didn't want to make the test so big and explicit